library/std/src/io/error.rs

```diff
@@ -400,6 +400,11 @@ pub enum ErrorKind {
     #[stable(feature = "out_of_memory_error", since = "1.54.0")]
     OutOfMemory,
 
+    /// The operation was partially sucessful and needs to be checked
+    /// later on due to not blocking
+    #[unstable(feature = "io_error_inprogress", issue = "none")]
```

Comment:
this is the only place where the attribute is needed, remove the others

Reply:
Done